gtsam/hybrid/HybridGaussianFactorGraph.cpp

```diff
@@ -195,7 +208,10 @@ hybridElimination(const HybridGaussianFactorGraph &factors,
     if (graph.empty()) {
       return {nullptr, nullptr};
     }
-    auto result = EliminatePreferCholesky(graph, frontalKeys);
```

Comment:
I think auto will just work since we are on GCC7

Reply:
I prefer the readability. Made debugging easier.